site/cypress/e2e/Cypress-UI/pronouns_change.spec.js

```diff
@@ -137,7 +137,7 @@ describe('Tests cases abut changing user pronouns', () => {
         cy.contains('Test Anonymous thread, should not show pronouns').find('.post_user_id').click();
         cy.get('.post_user_id').should('contain', 'Anonymous');
         cy.get('.post_user_pronouns').should('not.exist');
-        cy.get('.first_post').find('.fa-trash').click();
+        cy.get('.first_post > .post-action-container > .dropdown-menu ').find(':contains("Delete")').click({force: true});
```

Comment:
Is there a reason for these changes?

Reply:
Yes, since there is no 'fa-trash' icon, the delete button is wrapped in the dropdown menu.